src/main/java/edu/kis/powp/jobs2d/drivers/DriverComposite.java

```diff
@@ -0,0 +1,48 @@
+package edu.kis.powp.jobs2d.drivers;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DriverComposite implements Job2dDriver {
+    private List<Job2dDriver> driverList;
```

Comment:
Original project formatting convention used tabs (should not be changed to spaces).